### DISCUSSION TITLE
Cant login as admin after install

### DISCUSSION THREAD

**Discussion by buism** [photoprism/photoprism#1547] [Q&A] [answered]
Installed on Unraid docker. Set simple initial admin password.
When i try and login webgui it says invalid credentials.
trying "admin" with simple pw
any ideas?

**Comment by lastzero** [comment, marked as answer]
No, it was not a bad idea (the minimum length is 4 characters). Note the variable is only for setting the INITIAL password. So when you've already started PhotoPrism once, you can't change the password anymore like this. Instead you either need to start with a fresh database or run the `photoprism passwd` command in a terminal.
